public/app/features/trails/MetricSelect/AddToExplorationsButton.tsx

```diff
@@ -0,0 +1,139 @@
+import { DataFrame, TimeRange } from '@grafana/data';
+import { usePluginLinks } from '@grafana/runtime';
+import { SceneComponentProps, sceneGraph, SceneObjectBase, SceneObjectState, SceneQueryRunner } from '@grafana/scenes';
+import { DataQuery, DataSourceRef } from '@grafana/schema';
+import { IconButton } from '@grafana/ui';
+
+import MimirLogo from '../../../plugins/datasource/prometheus/img/mimir_logo.svg';
```

Comment:
Would be nice to have this not coupled to Grafana :)

Reply:
Totally, this is one more thing that will need decoupling before Explore Metrics is externalized.